### PR TITLE
Feature/welcome page

### PR DESCRIPTION
**Pull request by adlamas**
- Created form for welcome page
- Created anchors

[Trello card](https://trello.com/c/NQdirui1/9-5-as-a-user-i-should-be-able-to-sign-up-so-my-account-is-created-in-target-mvd)

Captura de Pantalla 2023-01-17 a la(s) 18 45 57

### CODE REVIEW

File: src/components/signUp/Form.tsx [hunk 1]
```diff
@@ -35,11 +38,12 @@ const Form = () => {
...
-        accessToken: res.headers['access-token'],
-        client: res.headers['client'],
-        uid: res.headers['uid']
-      }
+      //return {
+      //  accessToken: res.headers['access-token'],
+      //  client: res.headers['client'],
+      //  uid: res.headers['uid']
+      //}
+      nav("/");
```
Comment (tobicorradi) [inline src/components/signUp/Form.tsx:46]: WDYT about taking this `/` an save it as a constant?

File: src/index.tsx [hunk 2]
```diff
@@ -13,6 +14,7 @@ root.render(
   <React.StrictMode>
     <BrowserRouter>
       <Routes>
+        <Route path="/" element={<SignIn />} />
```
Comment (tobicorradi) [inline src/index.tsx:17]: Then you can replace the `/` string with that same constant that I mentioned above. It keeps everything more simple. WDYT?

File: src/components/signIn/Form.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        <h5 className="main-page--description">
+          Create a  target  wherever on the map, specify your interest: Travel,
+          Dating, Music, etc and start conecting with others who share your interest.
+        </h5>
+      </p>     
+
+      <section className='main-page--input-section'>
+        <div className="main-page--input-div">
+          <label htmlFor="email" className='main-page--input-label'>EMAIL</label>
+          <input style={{}}
```
Comment (tobicorradi) [inline src/components/signIn/Form.tsx:33]: Do you need that empty `style={{}}`?

File: src/components/SignIn.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import menu from 'assets/images/menu.png'
+import SignInForm from 'components/signIn/Form'
+import RightSection from 'components/common/RightSection'
+
+function SignIn() {
```
Comment (tobicorradi) [inline src/components/SignIn.tsx:5]: Are you going to use 
``````
function SignIn () {
``````
or
``````
const SignIn = () => 
``````

for every component? Remember that it's a good practice to have consistency.

**Review by tobicorradi — APPROVED** [review]
LGTM, left a non-blocking suggestions

File: src/components/signIn/Form.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  const [password, setPassword] = useState("");
+
+  return (
+    <form onSubmit={e => {
+      e.preventDefault();
+    }}
+      className='main-page-form'
+    >
+      <div className=''>
+        <img src={smiliesIcon} alt="Two faces smiling" />
```
Comment (bmarcio20) [inline src/components/signIn/Form.tsx:19]: Awesome that you kept A11Y in mind!

File: src/components/signIn/Form.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+          <label htmlFor="password" className='main-page--input-label'>PASSWORD</label>
+          <input
+            type="password"
+            name="password"
+            placeholder="MIN. 6 CHARACTERS LONG"
+            onChange={e => {
+              setPassword(e.target.value);
+            }}
+          />
+        </div>
```
Comment (bmarcio20) [inline src/components/signIn/Form.tsx:52]: This is not blocking at all... but remind me to discuss this section through slack afterwards. (I don't want to spoil it here :P )

File: src/components/signUp/Form.css [hunk 7]
```diff
@@ -79,7 +88,7 @@ body {
   height: 2.78em;
   width: 13.5em;
   font-size: 0.65em;
-  text-align: center;
+  margin: 0;
```
[the remark this answers is not in the document]
Comment (bmarcio20) [reply src/components/signUp/Form.css:91]: That is a good piece of advice! I usually do the same on my projects.

**Review by bmarcio20 — APPROVED** [review]
I left some non-blocking comments. But might be worth to address some of the comments before merging this PR.

File: src/components/SignIn.tsx (hunk 4 above)
Comment (adlamas) [reply src/components/SignIn.tsx:5]: Probably I created this before what we talked 😋

File: src/components/signIn/Form.css [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.sign-in-text {
+  border-top: var(--sign-in-border);
+  font-weight: 500;
+  font-size: .7em;
+  text-align: center;
+  padding-top: .7em;
+  width: 10.7em
+}
```
[the remark this answers is not in the document]
Comment (adlamas) [reply src/components/signIn/Form.css:94]: No, it's ok, happens that I am not used, yet, to maintain the convention

**Pull request merged**